lib/alertwildfire_API.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018 The Fuego Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+import sys
+fuegoRoot = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, os.path.join(fuegoRoot, 'lib'))
+sys.path.insert(0, fuegoRoot)
+import settings
+settings.fuegoRoot = fuegoRoot
+import datetime
+import requests
+import time
+import img_archive
+
+
+
+if settings.alertwildfirekey == 'do not put real key in files in open source git repo':
+    logging.warning('Please update settings.alertwildfirekey with the key provided to you')
+
+
+def get_all_camera_info():
+    headers = {'X-Api-Key': settings.alertwildfirekey}
+    response = requests.get('https://data.alertwildfire.org/api/firecams/v0/cameras', headers=headers)
+    if response.status_code == 404:
+        return 
+    exec('listofcameras = '+response.text, locals(),globals())
+    return listofcameras
+
+def get_individual_camera_info(cameraID):
+
+    headers = {'X-Api-Key': settings.alertwildfirekey}
+    response = requests.get('https://data.alertwildfire.org/api/firecams/v0/cameras'+'?name='+cameraID, headers=headers)
```

Comment:
Please use urllib.parse.urlencode to construct the URL query params.
https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode